specification/eventgrid/data-plane/Microsoft.ResourceNotifications/stable/2018-01-01/HealthResources.json

```diff
@@ -0,0 +1,93 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-01-01",
+    "title": "Schema of Azure Resource Management events published to Azure Event Grid via Azure Resource Notifications",
+    "description": "Describes the schema of the Azure Resource Management events published to Azure Event Grid. This corresponds to the Data property of an EventGridEvent."
+  },
+  "paths": {},
+  "definitions": {
+    "HealthResourcesAvailabilityStatusChangedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.ResourceNotifications.HealthResources.AvailabilityStatusChanged event.",
+      "type": "object",
+      "properties": {
+        "resourceInfo": {
+          "description": "Data specific to the resource",
+          "$ref": "#/definitions/ResourceInfoForAvailabilityStatusChangedEvent"
+        },
+        "apiVersion": {
+          "description": "api version of the resource payload",
+          "type": "string"
+        }
+      },
+      "x-ms-examples": {
+        "HealthResourcesAvailabilityStatusChangedEventGridSchema": {
+          "$ref": "./examples/event-grid-schema/HealthResources_AvailabilityStatusChanged.json"
+        }
+      }
+    },
+    "ResourceInfoForAvailabilityStatusChangedEvent": {
```

Comment:
Rather than define a nested model like this, it is more typical to have a base event model that the other models would inherit from. Here is an example - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/eventgrid/data-plane/Microsoft.AppConfiguration/stable/2018-01-01/AppConfiguration.json#L54-L88